hack/verify-shellcheck.sh

```diff
@@ -39,6 +39,10 @@ disabled=(
   # this lint disallows non-constant source, which we use extensively without
   # any known bugs
   1090
+  # this lint warns when shellcheck cannot find a sourced file
+  # this wouldn't be a bad idea to warn on, but it fails on lots of path
+  # dependent sourcing, so just disable enforcing it
+  1091
```

Comment:
might need to drop this one, I'm not sure if this is a new lint or newly enhanced lint.

in either case, it's not so much a lint as a warning about the linter not being able to follow some `source`d files even though we've told it to try. that might be useful to know but should not block CI.

Reply:
Not an issue